docs/REST API/rest-api.md

```diff
@@ -8711,6 +8905,8 @@ Get a list of all software.
 | min_cvss_score | integer | query | _Available in Fleet Premium_. Filters to include only software with vulnerabilities that have a CVSS version 3.x base score higher than the specified value.   |
 | max_cvss_score | integer | query | _Available in Fleet Premium_. Filters to only include software with vulnerabilities that have a CVSS version 3.x base score lower than what's specified.   |
 | exploit | boolean | query | _Available in Fleet Premium_. If `true`, filters to only include software with vulnerabilities that have been actively exploited in the wild (`cisa_known_exploit: true`). Default is `false`.  |
+| platform | string | query | Filter software by platform. Supported values are `darwind`, `windows`, and `linux`.  |
```

Comment:
@dantecatalfamo Could you double-check which platforms can be specified? We decided to document the `platform` filter for this endpoint.